packages/sdk/src/light_push/light_push.ts

```diff
@@ -84,17 +81,6 @@ export class LightPush implements ILightPush {
 
     log.info("send: attempting to send a message to pubsubTopic:", pubsubTopic);
 
-    if (!this.connectionManager.isTopicConfigured(pubsubTopic)) {
```

Comment:
I think we should replace this with check for `cluster`, if the one from `encoder` is supported by underlying node
Same for `Filter` in `@waku/sdk`

Reply:
I am not convinced this is the right place, I am fixing this check in the connection manager in my follow-up PR

Reply:
True, I forgot about this feature of `peerManager`

Reply:
True, I forgot about this feature of `peerManager`

So in short, users will get `NO_PEERS` error code instead of `TOPIC_MISCONFIGURED`.
I think that is enough as we are aiming for simplicity in the code and we will try to improve it later

Reply:
maybe worth adding e2e and / or unit test

Reply:
Will add before merging.

Reply:
We already have https://github.com/waku-org/js-waku/blob/c7682ea67c54d2c26a68ce96208003fb1ffc915c/packages/sdk/src/peer_manager/peer_manager.spec.ts#L167 to test `getPeers` so not sure additional test make sense. Especially that `light_push.spec.ts` mocks the peer manager.